q2_viromics/tests/test_format.py

```diff
@@ -56,6 +59,63 @@ def test_HMMFormat_neg1(self):
             format.validate()
 
     def test_CheckVDBDirFmt(self):
-        filepath = self.get_data_path("type/checkVdb/")
+        filepath = self.get_data_path("type/db/")
         format = CheckVDBDirFmt(filepath, mode="r")
+
         format.validate()
+
+
+class TestCheckVDBDirFmtPathMakers(unittest.TestCase):
+    package = "q2_viromics.tests"
+
+    def setUp(self):
+        self.format = CheckVDBDirFmt("/dummy/path", mode="r")
+
+    @patch.object(CheckVDBDirFmt, "hmm_files_path_maker")
+    def test_hmm_files_path_maker(self, mock_hmm_files):
+        mock_hmm_files.return_value = "/dummy/path/hmm/dir/name.hmm"
+        result = self.format.hmm_files_path_maker("outer", "dir", "name")
+        mock_hmm_files.assert_called_once_with("outer", "dir", "name")
+        self.assertEqual(result, "/dummy/path/hmm/dir/name.hmm")
```

Comment:
You have to explicitly state the parameters
```suggestion
    @patch.object(CheckVDBDirFmt, "hmm_files_path_maker")
    def test_hmm_files_path_maker(self, mock_hmm_files):
        mock_hmm_files.return_value = "/dummy/path/hmm/dir/name.hmm"
        result = self.format.hmm_files_path_maker(outer="outer", dir="dir", name="name")
        mock_hmm_files.assert_called_once_with("outer", "dir", "name")
        self.assertEqual(result, "/dummy/path/hmm/dir/name.hmm")
```